t/aws.t

```diff
@@ -33,13 +33,8 @@ for (<{get,post}-*>) {
 
     ( $path, my $query ) = split /\?/, $path;
 
-    my %query;
-
-    for ( split /&/, $query // '' ) {
-        my ( $k, $v ) = split /=/;
-
-        push @{ $query{$k} }, $v;
-    }
+    my @query = split /&/, ( $query // '' );
```

Comment:
do we need to worry about uri encoding?

Reply:
Worry about it how?

These are coming from already stringified requests. Here's an example: https://github.com/cv-library/WebService-S3-Tiny/blob/master/t/aws/get-vanilla-query-order-value/get-vanilla-query-order-value.req#L1

Truth be told, this doesn't need to be a super general-purpose bit of code. We are only dealing with a limited set of test cases that were developed by Amazon themselves, and they don't contain anything really crazy from what I can tell.